configuration/configure-proxy.adoc

```diff
@@ -0,0 +1,23 @@
+[id="configure-proxy"]
+= Configuring a proxy for external network access
+include::modules/common-attributes.adoc[]
+:context: configure-proxy
+
+toc::[]
+
+[role="_abstract"]
+If your network configuration restricts outbound traffic through proxies, you can configure proxy settings in {product-title} to route traffic through a proxy.
+
+When you use a proxy with {product-title}:
+
+* All outgoing HTTP, HTTPS, and other TCP traffic from Central and Scanner goes through the proxy.
+* Traffic between Central and Scanner does not go through the proxy.
+* The proxy configuration does not affect the other {product-title} components.
+* When you are not using the offline mode, and a Collector running in a secured cluster needs to download an additional kernel module or eBPF probe at runtime:
+** The collector attempts to download them by contacting Sensor.
+** The Sensor then forwards this request to Central.
+** Central uses the proxy to locate the module or probe at `https://collector-modules.stackrox.io`.
```

Comment:
Is this a URL that users are supposed to be able to reach? If so, I get an "Access defined" message. If not, can you escape it so that it isn't clickable?